src/OrbitGl/TimeGraph.h

```diff
@@ -219,6 +223,9 @@ class TimeGraph : public orbit_gl::CaptureViewElement, public orbit_gl::Timeline
   orbit_gl::GlSlider* last_mouseover_slider_ = nullptr;
   std::unique_ptr<orbit_gl::TrackContainer> track_container_;
   std::unique_ptr<orbit_gl::TimelineUi> timeline_ui_;
+  // A shared ptr is needed for the buttons because they are pickable.
```

Comment:
I don't understand the relationship between being pickable and necessitating a shared_ptr, could you elaborate? This seems counterintuitive. Is there a situation where the button could outlive the TimeGraph? Seems like we should fix that instead, no? Why not use a std::unique_ptr and pass raw-pointers for picking? The problem I'm seeing here is that even though the button will be guaranteed to be valid by using a shared_ptr, its real owner might get deleted and we'll now try to perform an action on the deleted owner.

Reply:
I will point to a previous PR small discussion related to Track Header: https://github.com/google/orbit/pull/3491#discussion_r842713844

I also don't like it, but this should be a change independent of this PR. I would like to have input from @reichlfl